modules/dns_common/dns_common.rb

```diff
@@ -170,10 +170,10 @@ def aaaa_record_conflicts(fqdn, ip)
       end
     end
 
-    def cname_record_conflicts(fqdn, target)
-      current = resolver.getresources(fqdn, Resolv::DNS::Resource::IN::CNAME)
+    def cname_record_conflicts(fqdn, host_alias)
+      current = resolver.getresources(host_alias, Resolv::DNS::Resource::IN::CNAME)
```

Comment:
I get your point. Thanks for clarifying. 
So, should we standardize this or leave as it is?